src/exchange/CeloNewsFeedItem.tsx

```diff
@@ -51,6 +53,28 @@ export default function CeloNewsFeedItem({ article, testID }: Props) {
   )
 }
 
+// This is a skeleton placeholder for when the feed is loading
+// It's a simplified version of the real component, reusing styles where possible
+CeloNewsFeedItem.Skeleton = () => (
+  <SkeletonPlaceholder
+    borderRadius={4}
+    backgroundColor={colors.gray2}
+    highlightColor={colors.white}
+    testID="CeloNewsFeedItemSkeleton"
+  >
+    <View style={styles.contentContainer}>
+      <View style={{ ...styles.author, width: 60 }} />
+      <View style={styles.row}>
+        <View style={{ flex: 1 }}>
+          <View style={{ ...styles.title, flex: undefined, marginBottom: 4 }} />
+          <View style={{ ...styles.title, flex: undefined, width: '60%' }} />
+        </View>
+        <View style={styles.image} />
+      </View>
+    </View>
```

Comment:
I broke our rule of not using inlines styles here because I thought it made more sense to have it directly there.
But let me know if you think I should really not do this.